core/chains/evm/txmgr/broadcaster_test.go

```diff
@@ -1524,7 +1524,7 @@ func TestEthBroadcaster_ProcessUnstartedEthTxs_Errors(t *testing.T) {
 		pgtest.MustExec(t, db, `DELETE FROM evm.txes WHERE nonce = $1`, localNextNonce)
 	})
 
-	t.Run("eth tx is left in progress if eth node returns insufficient eth", func(t *testing.T) {
+	t.Run("if eth node returns insufficient eth, eth tx is replaced with new re-estimated tx, and keep in progress", func(t *testing.T) {
```

Comment:
Let's have it more aligned to the old message
```suggestion
	t.Run("tx is left in progress and its attempt gets replaced with a new re-estimated attempt if node returns insufficient eth", func(t *testing.T) {
```